tests/wpt/metadata/MANIFEST.json

```diff
@@ -397231,6 +397291,54 @@
    "b960bef807da94c0146ed2f537eaa1e05ec9a0ab",
    "testharness"
   ],
+  "_certs/064162.pem": [
```

Comment:
The update-manifest script isn't smart enough to ignore files that it shouldn't include, so you'll need to remove all of these extraneous lines manually. Sorry, I find this behavior really annoying too!